MAUI/DataGrid/scrolling.md

```diff
@@ -48,4 +48,41 @@ You can scroll programmatically to a particular column using the `SfDataGrid.Scr
 {% highlight C# %}
 this.dataGrid.ScrollToColumnIndex(4, true, ScrollToPosition.MakeVisible);
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Scrollbar Visibility
+
+You can change the visibility of the horizontal and vertical scrollbar using the [SfDataGrid.HorizontalScrollBarVisibility]() and [SfDataGrid.VerticalScrollBarVisibility]() properties respectively. By default, the visibility of both the horizontal and vertical scrollbars is set to `Default`.
+
+{% tabs %}
+{% highlight xaml %}
+    <sfgrid:SfDataGrid x:Name="dataGrid"                                       
+                       ItemsSource="{Binding OrdersInfo}"         
+                       HorizontalScrollBarVisibility="Never"
+                       VerticalScrollBarVisibility="Never">   
+    </sfgrid:SfDataGrid> 
+{% endhighlight %}
+
+{% highlight C# %}
+namespace GettingStarted
```

Comment:
These codes will be enough.

dataGrid.HorizontalScrollBarVisibility = ScrollBarVisibility.Never;
dataGrid.VerticalScrollBarVisibility = ScrollBarVisibility.Never;

Reply:
updated